### PR TITLE
QS- Public Cloud - edit round 3

### PR DESCRIPTION
**Pull request by Loquacity**
@mbrookhuis and @rjschwei thanks for the feedback on https://github.com/SUSE/spacewalk/issues/10776. Here is the next edit round PR for you.

Note that I've taken the licence out for now, but will check to see what we need to do there. I'm hoping we can either leave it off, or use a shorter version.

PDF:
[suse_manager_quickstart-public-cloud_guide.pdf](https://github.com/uyuni-project/uyuni-docs/files/4406250/suse_manager_quickstart-public-cloud_guide.pdf)

### CODE REVIEW

File: modules/quickstart-public-cloud/pages/installation.adoc [hunk 1]
```diff
@@ -18,10 +18,10 @@ Follow the prompts, and wait for the setup to finish.
 
 For detailed instructions on setting up {productname} with {yast}, see xref:installation:server-setup.adoc[].
 
-Register your instance with {scc}, using this command:
+Register your instance with {scc}, using this command, specifying the registration number and the email address associated with it:
 
 ----
-sudo SUSEConnect -r <Registration_Number>
+sudo SUSEConnect -r <Registration_Number> -e <registered_email_address>
```
Comment (keichwa) [inline modules/quickstart-public-cloud/pages/installation.adoc:24]: the sudo way is not wrong but we usually say run this command as root. We probably should have this in the intro section of a book.

File: modules/quickstart-public-cloud/pages/installation.adoc [hunk 2]
```diff
@@ -57,6 +57,11 @@ The username and password varies depending on your provider.
 | Microsoft Azure       | ``admin``         |``<instance-name>-suma``
 |===
 
+
+When you log in to the administrator account for the first time, you will have an automatically generated organization name.
```
Comment (keichwa) [inline modules/quickstart-public-cloud/pages/installation.adoc:61]:
```suggestion
When you sign in to the administrator account for the first time, you will have an automatically generated organization name.
```

File: modules/quickstart-public-cloud/pages/installation.adoc [hunk 3]
```diff
@@ -57,6 +57,11 @@ The username and password varies depending on your provider.
...
+Change this by navigating to menu:Admin[Organizations], and editing the organization name.
+
+
 [IMPORTANT]
 ====
 When you have logged in to the administrator account for the first time, change the default password to protect your account.
```
Comment (keichwa) [inline modules/quickstart-public-cloud/pages/installation.adoc:67]:
```suggestion
When you have signed in to the administrator account for the first time, change the default password to protect your account.
```
drive-by edit.  Wondering why we first talk about changing the org name and then doing the important password change.

File: modules/quickstart-public-cloud/pages/setup.adoc [hunk 4]
```diff
@@ -34,18 +23,20 @@ For security and access settings, you will need to enable HTTPS access.
...
+In GCE, check the ``Allow HTTPS traffic`` box under the ``Firewall`` section.
+
+[WARNING]
+====
+When running {productname} on public clouds, make sure you apply security measures to limit access to the right people.
+A world-accessible {productname} would give access to your infrastructure to anyone, and could breach your {suse} support agreement.
+====
 
 For storage settings, make sure you have at least 100GB.
 We also recommend that the repositories and the database for {productname} are stored on a separate storage volume.
```
Comment (keichwa) [inline modules/quickstart-public-cloud/pages/setup.adoc:35]:
```suggestion
We also recommend that the repositories and the database for {productname} are stored on separate storage volumes.
```
Elsewhere you use plural.  And "devices" instead of "volumes".

**Comment by Loquacity** [comment]
Merging this one now, so we can prepare for publication in beta 3.
